ga4gh/backend.py

```diff
@@ -9,6 +9,7 @@
 import os
 
 import ga4gh.protocol as protocol
+from ga4gh.frontend import app
```

Comment:
Beacon 0.2 includes a bunch of fields that are probably best set in a config file. I cringed when I wrote this line, but I needed the app in order to get to the config.  Is there a better an alternative way  to access to app.config or a better design I could use?